autocomplete.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+func suggestTokensFromInputting(focus []rune, pos int) [][]rune {
+	return suggestInputting(focus, pos, [][]rune{
+		[]rune("select"),
```

Comment:
We need to declare the reserved words as constants on lexical package to use these there and here. 
This pull request is too big already so please, let us work in another PR.